server/shared/src/main/scala/org/http4s/server/websocket/WebSocketBuilder.scala

```diff
@@ -41,7 +44,36 @@ final case class WebSocketBuilder[F[_]: Applicative](
     onHandshakeFailure: F[Response[F]],
     onClose: F[Unit],
     filterPingPongs: Boolean,
-) {
+    autoPing: Option[(FiniteDuration, WebSocketFrame.Ping)],
```

Comment:
Why did we change this one? It's deprecated.

Reply:
Reverted.